demos/smart_framing_demo/cpp_gapi/smart_framing_demo_gapi.hpp

```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+#pragma once
+
+#include <gflags/gflags.h>
+#include <utils/default_flags.hpp>
+#include <models/detection_model.h>
```

Comment:
Transitive includes is a bad thing, please include it in file where you actualy use it.

Reply:
Done

Reply:
Done